cmd/juno/main.go

```diff
@@ -99,6 +99,7 @@ func newRootCmd() *cobra.Command {
 	// Log
 	rootCmd.PersistentFlags().StringVar(&cfg.Log.Level, "log-level", "info", "verbosity of the logs. Options: debug, info, warn, error, dpanic, panic, fatal.")
 	rootCmd.PersistentFlags().BoolVar(&cfg.Log.Json, "log-json", false, "print logs in json format. Useful for automated processing. Typically omitted if logs are only viewed from console.")
+	rootCmd.PersistentFlags().BoolVar(&cfg.Log.Color, "no-color", false, "disable colour coded logs. Colour coded logs are enabled by default.")
```

Comment:
To maintain the naming convention for CLI flags, let's change to the name from `no-color` to `log-color` and set the default value to `true` (and modify the usage statement to reflect that). This way the UI is consistent for the CLI params, environment variables, and config file.

Reply:
Thanks! Got it.